### ISSUE TITLE
NameError: name 'Image' is not defined

### ISSUE THREAD

**Issue by rajutechie** [TomSchimansky/CustomTkinter#706]
<img width="718" alt="Screenshot 2022-11-30 at 9 44 32 PM" src="https://user-images.githubusercontent.com/68769797/204945924-3358df0f-ee7a-48b6-93c5-ae15acff04ae.png">
Hello, 

Please find the below code snippet
```
import tkinter
import customtkinter  

root_tk = tkinter.Tk()  
root_tk.geometry("400x240")
root_tk.title("CustomTkinter Test")


def button_function():
    print("button pressed")

button = customtkinter.CTkButton(master=root_tk, corner_radius=10, command=button_function)
button.place(relx=0.5, rely=0.5, anchor=tkinter.CENTER)

root_tk.mainloop()
```

I am getting below error

```
 /Users/MyPc/Documents/Workspace/Python_Pocs/Downloader/venv/bin/python /Users/MyPc/Documents/Workspace/Python_Pocs/Downloader/main.py 
Traceback (most recent call last):
  File "/Users/MyPc/Documents/Workspace/Python_Pocs/Downloader/main.py", line 2, in <module>
    import customtkinter  # <- import the CustomTkinter module
  File "/Users/MyPc/Documents/Workspace/Python_Pocs/Downloader/venv/lib/python3.10/site-packages/customtkinter/__init__.py", line 10, in <module>
    from .windows.widgets.appearance_mode import AppearanceModeTracker
  File "/Users/MyPc/Documents/Workspace/Python_Pocs/Downloader/venv/lib/python3.10/site-packages/customtkinter/windows/__init__.py", line 1, in <module>
    from .ctk_tk import CTk
  File "/Users/MyPc/Documents/Workspace/Python_Pocs/Downloader/venv/lib/python3.10/site-packages/customtkinter/windows/ctk_tk.py", line 9, in <module>
    from .widgets.theme import ThemeManager
  File "/Users/MyPc/Documents/Workspace/Python_Pocs/Downloader/venv/lib/python3.10/site-packages/customtkinter/windows/widgets/__init__.py", line 1, in <module>
    from .ctk_button import CTkButton
  File "/Users/MyPc/Documents/Workspace/Python_Pocs/Downloader/venv/lib/python3.10/site-packages/customtkinter/windows/widgets/ctk_button.py", line 8, in <module>
    from .core_widget_classes import CTkBaseClass
  File "/Users/MyPc/Documents/Workspace/Python_Pocs/Downloader/venv/lib/python3.10/site-packages/customtkinter/windows/widgets/core_widget_classes/__init__.py", line 2, in <module>
    from .ctk_base_class import CTkBaseClass
  File "/Users/MyPc/Documents/Workspace/Python_Pocs/Downloader/venv/lib/python3.10/site-packages/customtkinter/windows/widgets/core_widget_classes/ctk_base_class.py", line 18, in <module>
    from ..image import CTkImage
  File "/Users/MyPc/Documents/Workspace/Python_Pocs/Downloader/venv/lib/python3.10/site-packages/customtkinter/windows/widgets/image/__init__.py", line 1, in <module>
    from .ctk_image import CTkImage
  File "/Users/MyPc/Documents/Workspace/Python_Pocs/Downloader/venv/lib/python3.10/site-packages/customtkinter/windows/widgets/image/ctk_image.py", line 8, in <module>
    class CTkImage:
  File "/Users/MyPc/Documents/Workspace/Python_Pocs/Downloader/venv/lib/python3.10/site-packages/customtkinter/windows/widgets/image/ctk_image.py", line 22, in CTkImage
    light_image: Image.Image = None,
NameError: name 'Image' is not defined

Process finished with exit code 1
```

> I am using Mac Ventura M1, Python 3.10
Why I am getting the above error?

**Comment by Michelzxc** [comment]
You need install pillow module. CTkImage need pillow. Try install in your environment: python -m pip install pillow

**Comment by DanPos2506** [comment]
Having this issue too, installed pillow and uninstalled 5.0.0 and installed 4.6.3 and not working for me. Full error attached:

> "G:\My Drive\Programming\Python\CustomTKInterExamples\CustomTkinter\venv\Scripts\python.exe" "G:\My Drive\Programming\Python\CustomTKInterExamples\CustomTkinter\examples\simple_example.py" 
Traceback (most recent call last):
  File "G:\My Drive\Programming\Python\CustomTKInterExamples\CustomTkinter\examples\simple_example.py", line 2, in <module>
    import customtkinter
  File "G:\My Drive\Programming\Python\CustomTKInterExamples\CustomTkinter\customtkinter\__init__.py", line 10, in <module>
    from .windows.widgets.appearance_mode import AppearanceModeTracker
  File "G:\My Drive\Programming\Python\CustomTKInterExamples\CustomTkinter\customtkinter\windows\__init__.py", line 1, in <module>
    from .ctk_tk import CTk
  File "G:\My Drive\Programming\Python\CustomTKInterExamples\CustomTkinter\customtkinter\windows\ctk_tk.py", line 9, in <module>
    from .widgets.theme import ThemeManager
  File "G:\My Drive\Programming\Python\CustomTKInterExamples\CustomTkinter\customtkinter\windows\widgets\__init__.py", line 1, in <module>
    from .ctk_button import CTkButton
  File "G:\My Drive\Programming\Python\CustomTKInterExamples\CustomTkinter\customtkinter\windows\widgets\ctk_button.py", line 8, in <module>
    from .core_widget_classes import CTkBaseClass
  File "G:\My Drive\Programming\Python\CustomTKInterExamples\CustomTkinter\customtkinter\windows\widgets\core_widget_classes\__init__.py", line 2, in <module>
    from .ctk_base_class import CTkBaseClass
  File "G:\My Drive\Programming\Python\CustomTKInterExamples\CustomTkinter\customtkinter\windows\widgets\core_widget_classes\ctk_base_class.py", line 18, in <module>
    from ..image import CTkImage
  File "G:\My Drive\Programming\Python\CustomTKInterExamples\CustomTkinter\customtkinter\windows\widgets\image\__init__.py", line 1, in <module>
    from .ctk_image import CTkImage
  File "G:\My Drive\Programming\Python\CustomTKInterExamples\CustomTkinter\customtkinter\windows\widgets\image\ctk_image.py", line 8, in <module>
    class CTkImage:
  File "G:\My Drive\Programming\Python\CustomTKInterExamples\CustomTkinter\customtkinter\windows\widgets\image\ctk_image.py", line 22, in CTkImage
    light_image: Image.Image = None,
NameError: name 'Image' is not defined

Process finished with exit code 1

**Comment by TomSchimansky** [comment]
should be resolved now with version 5.0.1

**Comment by rajutechie** [comment]
Verified and the above issue got resolved in 5.0.1 version.
